tests/dbshell/test_mysql.py

```diff
@@ -122,13 +116,14 @@ def get_command_line_arguments(self, connection_settings, parameters=None):
     def test_charset_configured(self):
         self.assertEqual(
             ['mysql', '--user=someuser', '--password=somepassword',
-             '--host=somehost', '--port=444', 'somedbname', '--default-character-set=utf8'],
```

Comment:
This is what we want to test. Please restore `'--default-character-set=utf8'` in this line.